.github/workflows/ci.yml

```
@@ -0,0 +1,148 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+
+  linux:
+
+    strategy:
+      fail-fast: true
```

```suggestion
      fail-fast: false
```
Although it might be good if you are quickly debugging, I think this might slow down getting insight (resulting in more runs)